person.rb

```diff
@@ -20,6 +21,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+    rental.person = self unless rental.person == self
+  end
+
```

Comment:
- Hey, let's ensure the person itself is sent as a parameter to create the new rental achieving the 'has-many' association


```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```